### PR TITLE
Adds ability to change vault-helper url in tarmak.yaml

### LINKED ISSUE #393: Customize vault-helper download URL
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind feature

**What you expected to happen**:
It would be useful to allow the override of the `vault-helper` download url in vault_client module so it can be downloaded from local mirrors instead of github.com

### PR DESCRIPTION
**Pull request by JoshVanL**
**What this PR does / why we need it**:
Adds option to set a custom vault-helper download url. Still defaults to normal github location without the option set.

fixes #393 

```release-note
Adds custom vault-helper URL option in tarmak.yaml
```

### CODE REVIEW

File: docs/user-guide.rst [hunk 1]
```diff
@@ -844,6 +844,23 @@ make sure you don't define any volumes with the name ``docker``.
...
+
+It is possible to configure the download URL of vault-helper to something other
+than the default github store. To make this cluster wide change, add the custom
+URL to the ``url`` attribute, under the ``vaultHelper`` header in the cluster:
+
+.. code-block:: yaml
+
+  clusters:
+  - amazon:
+      ebsEncrypted: true
```
[the remark this answers is not in the document]
Comment (JoshVanL) [reply docs/user-guide.rst]: @simonswine In what way?

[the remark this answers is not in the document]
Comment (simonswine) [reply docs/user-guide.rst]: Why do you have the amazon.ebsEncrypted there, it has nothing to do with vault-helper and might confuse people

**Comment by simonswine** [comment]
Thanks @JoshVanL
/approve
/lgtm

**Pull request merged**